ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -764,3 +769,10 @@ Compile And Query LLM model
           ...    json_body=${body}    json_header=${header}
           ...    insecure=${TRUE}
     END
+
+Run Install Script
+    ${rc}=    Run And Return Rc    git clone https://github.com/opendatahub-io/caikit-tgis-serving
+    Should Be Equal As Integers    ${rc}    ${0}
+    ${rc}=    Run And Watch Command    TARGET_OPERATOR=rhods CHECK_UWM=false ./scripts/install/kserve-install.sh
+    ...    cwd=caikit-tgis-serving/demo/kserve
```

Comment:
Given
```the script tries to install RHODS, but if it is already installed, it shouldn't break it (it's not optimal, but it is thought as internal tool)```
Can you add a check on RHODS being present (`Is RHODS Installed`) before this and abort the execution if it is?

Reply:
nope. I will try raising a PR for the instal script in order to make the RHODS deployment optional

Reply:
Yeah, that's the point :D since there's a risk it would mess up the installation if already present, and since we don't control the script directly, I would prefer to have this run only on a clean cluster. 
You could also trigger an Uninstall before proceeding with execution, but that complicates the flow quite a bit.

Reply:
let me see what I can do. However consider that this is not the default option. BY default the tests will use step-by-step installation which is implemented in RF

Reply:
Then why not just make it Fail if RHODS is already installed, given that it is an optional flow?

Reply:
because I expect RHODS installed in most of the cases. Think of a jenkins job which installs rhods but then failed for other reasons like connection issues or RHODS not installed by the jenkins timeout etc. Then you could not re-run the test because if would fail

Reply:
But in this specific case this script would be the one installing RHODS, so you would run it in a clean cluster and you wouldn't enable rhods installation.
If the script fails to install RHODS then the test run should also fail, since that's part of what this is trying to do, no?

Reply:
>But in this specific case this script would be the one installing RHODS,

I'm assuming that we may be running the tests in a cluster where RHODS is already installed or installed via jenkins. In this PR i'm not making our RF test to set the RHODS version to install but the script will try to install always rhods from operatorhub. So if you want a different version you'll need to install it differently.

It's not ideal neither case

Reply:
I understand it's not ideal, but AFAIK the documentation will point to the script and will tell users to run it, so IMHO we should be running it and expecting it work with its own install.
The only correct option here would be to remove the installation steps from the script, but I don't know if that's feasible / something we want to do

Reply:
fyi: https://github.com/opendatahub-io/caikit-tgis-serving/pull/91 
PR to make the rhods deployment optional in the install script